guide/pitchslider.svg

```diff
@@ -0,0 +1,1042 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
On all of your SVG files:
(1) Please crop the SVG width and height to match the width and height of your artwork (there is a tool for this in Inkscape under the File menu).
(2) Please save as "simple SVG" so as to remove all the Inkscape-specific fields that make the files much larger than they need to be.